Gerrit build fails with some baseline errors

See #131 gerrit build that is failed allthough there shouldn't be any problem:

https://ci.eclipse.org/jdt/job/eclipse.jdt.core-Github/job/PR-131/1/consoleFull

Gerrit complains about some errors which I don't understand:
```
10:25:23  [INFO] Adding repository https://download.eclipse.org/eclipse/updates/4.24-I-builds/I20220607-0700
10:25:29  [ERROR] Baseline and reactor have same fully qualified version, but with different content
10:25:29  different
10:25:29     .api_description: not present in baseline
10:25:29     .options: not present in baseline
10:25:29     META-INF/ECLIPSE_.RSA: different
10:25:29     META-INF/ECLIPSE_.SF: different
10:25:29     META-INF/MANIFEST.MF: different
```

Not sure why I20220607-0700 build is used at all, the last stable should be [I20220607-1940](https://download.eclipse.org/eclipse/downloads/drops4/I20220607-1940/).

@sravanlakkimsetti , @SDawley : do you have an idea what is missing?

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -32,7 +32,7 @@ pipeline {
 					
 					mvn -U clean verify --batch-mode --fail-at-end -Dmaven.repo.local=$WORKSPACE/.m2/repository \
 					-Pbuild-individual-bundles -Ptest-on-javase-17 -Pbree-libs -Papi-check \
-					-Djava.io.tmpdir=$WORKSPACE/tmp -Dcompare-version-with-baselines.skip=false -Dproject.build.sourceEncoding=UTF-8 \
+					-Djava.io.tmpdir=$WORKSPACE/tmp -Dproject.build.sourceEncoding=UTF-8 \
 					-Dtycho.surefire.argLine="--add-modules ALL-SYSTEM -Dcompliance=1.8,11,17 -Djdt.performance.asserts=disabled"
 					"""
 				}

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -32,7 +32,7 @@ pipeline {
 					
 					mvn -U clean verify --batch-mode --fail-at-end -Dmaven.repo.local=$WORKSPACE/.m2/repository \
 					-Pbuild-individual-bundles -Ptest-on-javase-17 -Pbree-libs -Papi-check \
-					-Djava.io.tmpdir=$WORKSPACE/tmp -Dcompare-version-with-baselines.skip=true -Dproject.build.sourceEncoding=UTF-8 \
+					-Djava.io.tmpdir=$WORKSPACE/tmp -Dcompare-version-with-baselines.skip=false -Dproject.build.sourceEncoding=UTF-8 \
 					-Dtycho.surefire.argLine="--add-modules ALL-SYSTEM -Dcompliance=1.8,11,17 -Djdt.performance.asserts=disabled"
 					"""
 				}

